cmd/logger.go

```diff
@@ -106,38 +95,58 @@ func (log *Logger) Printf(format string, args ...interface{}) {
 	}
 }
 
-// NewLogger - returns a new initialized logger.
-func NewLogger() *Logger {
-	logger := logrus.New()
-	logger.Out = ioutil.Discard
-	logger.Level = logrus.DebugLevel
-
-	l := &Logger{
-		logger:        logger,
-		consoleTarget: NewConsoleLogger(),
+func trimTrace(f string) string {
+	trimStrings := []string{
+		pathJoin(GOPATH, "/src/"),
+		pathJoin(GOROOT, "/src/"),
```

Comment:
Is `pathJoin()` MS Windows friendly?  You could use `filepath.Join()` which is in standard go library. `filepath.Join(GOROOT, "src") + string(filepath.Separator)` is more cleaner than `"/src/"`

Reply:
Done.